packages/entities/entities-plugins/src/components/free-form/shared/ArrayField.vue

```diff
@@ -9,18 +17,17 @@
   >
     <header class="ff-array-field-header">
       <KLabel
-        v-if="label"
         class="ff-array-field-label"
-        v-bind="labelAttributes"
-        :required="required"
+        v-bind="fieldAttrs"
+        :tooltip-attributes="fieldAttrs.labelAttributes.tooltipAttributes"
       >
-        {{ label }}
+        {{ fieldAttrs.label }}
         <template
-          v-if="labelAttributes?.info"
+          v-if="fieldAttrs.labelAttributes?.info"
           #tooltip
         >
           <!-- eslint-disable-next-line vue/no-v-html -->
-          <div v-html="labelAttributes?.info" />
+          <div v-html="fieldAttrs.labelAttributes.info" />
```

Comment:
Should we consider escaping the string here?

Also, I am wondering, maybe use a slot here for the tooltip to differentiate  with `tooltipAttributes` would have been nicer.

Reply:
Rendering HTML here is intended I think, because it's markdown in schema, we [parsed it to HTML](https://github.com/Kong/public-ui-components/blob/cdee76d8c708bf4275abdc8ce08ff154b8b3b810/packages/entities/entities-plugins/src/components/free-form/shared/composables.ts#L164) so it can be rendered correctly

Using a slot is a good idea, I'd wrap this `div` with a `<slot name="tooltip">`